benchmark_runner/common/elasticsearch/elasticsearch_operations.py

```diff
@@ -22,20 +25,37 @@ class ElasticSearchOperations:
     MAX_SEARCH_RESULTS = 1000
     MIN_SEARCH_RESULTS = 100
 
-    def __init__(self, es_host: str, es_port: str, es_user: str = '', es_password: str = '', timeout: int = 2000):
+    def __init__(self, es_host: str, es_port: str, es_user: str = '', es_password: str = '', es_ssl: bool = False, timeout: int = 2000):
         self.__es_host = es_host
         self.__es_port = es_port
         self.__es_user = es_user
         self.__es_password = es_password
+        self.__es_ssl = es_ssl
         self.__timeout = timeout
+
+        kwargs = {}
+        if self.__es_ssl:
+            es_password_http_prefix = "https://"
```

Comment:
elastic_http_prefix would be better (or store it in the environment_variables_dict).

Reply:
Do you meant to the variable name 'elastic_http_prefix' instead of 'es_password_http_prefix' ?

Reply:
Yes; it has nothing to do with passwords.

Reply:
All the variable of elastic are with es_ prefix. I think we can leave it for now. Do you agree ?

Reply:
Agreed, @kpouget  can you please change it ?

Reply:
I'm good with that.

Reply:
makes sense to me, I've updated the code

Reply:
Hi @kpouget, I still see 'es_ssl' as boolean parameter and not string. Can we change it to string, that the default will be http ?

Reply:
updated